docs/geedocs/answer/3230777.html

```diff
@@ -67,7 +67,7 @@
 <h3>Composite globes and maps</h3>
 <p>Portable globes and maps can also be assembled into layers to create a composite globe or map, a single file (<code>.glc</code>) that contains all the geospatial data for assembled layers. To assemble a composite file, you supply a KML polygon to cut your area of interest, then specify the layers and order in which they display. See <a href="../answer/4643041.html">Create composite globes and maps</a>.</p>
 <h2><a name="creating_a_portable_globe"></a>Create a portable globe or map</h2>
-<p>Cutting a globe or map is accomplished with a simple web interface. You use your mouse, or import KML, to define a polygon, which defines your 'area of interest.' This polygon not only defines the area that displays high-resolution imagery, but is also used by Fusion to create a localized search database.</p>
+<p>Cutting a globe or map is accomplished with a simple web interface. You import KML, or use your mouse, to define a polygon, which defines your 'area of interest.' This polygon not only defines the area that displays high-resolution imagery, but is also used by Fusion to create a localized search database.</p>
```

Comment:
I can change this one, but I can't promise what's used in the rest of the documentation. These quotes are unchanged from what they were before.